docs/4.x/tests-github.md

```diff
@@ -0,0 +1,54 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-travis
+---
+# Using GitHub Actions
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+## GitHub Actions we use
+
+Matomo uses GitHub Action to automatically run its build by various triggers.
+- [Build Tracker JS] Trigger by the comment `build js` into the pull request. That will compress js/piwik.js into matomo.js and piwik.js
+- [Build VUE] Auto trigger, it will build VUE file and commit back to the PR
+- [Composer Update] Auto trigger by cron，it will run `composer update` and commit back to the repo
+- [PHPCS check] Auto trigger, checking PHPCS code quality if not valid will return error details.
+- [Inactive PR] after 14 days, inactive PR will mark as state
+
+(New scripts should be documented in the list above.)
```

Comment:
can we add this to the `Create a new GitHub Action script` section as when you read the instructions for creating a new action, then you won't read this part and you'll forget to add it here